src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -1253,8 +1253,8 @@
             let groupingPath = $scope.selectedGrouping.path;
             let resetInclude = $scope.includeCheck;
             let resetExclude = $scope.excludeCheck;
-            var resetIncludeResult = [];
-            var resetExcludeResult = [];
```

Comment:
I remember this was set to var on purpose by the person who wrote this code. Can you check that resetting a grouping isn't broken from this change?